src/components/BulkEnrollmentPage/stepper/AddLearnersStep.test.jsx

```diff
@@ -33,31 +40,40 @@ const StepperWrapper = (props) => (
   </BulkEnrollContextProvider>
 );
 
+const flushPromises = () => new Promise(setImmediate);
```

Comment:
also TIL `setImmediate`

Reply:
Yeah, I *still* don't fully understand *why* this trick works, but it sure does work, so I've decided I don't care.